bftengine/src/bftengine/BFTEngine.cpp

```diff
@@ -1,24 +1,27 @@
 // Concord
 //
-// Copyright (c) 2018 VMware, Inc. All Rights Reserved.
+// Copyright (c) 2019 VMware, Inc. All Rights Reserved.
```

Comment:
nit - I believe this should be 2018-2019

Reply:
np